plugins/woo-ai/includes/class-woo-ai-deprecation-notice.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Class to handle deprecation notice tracking
+ *
+ * @package Woo_AI
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Woo_AI_Deprecation_Notice class
+ * Shows a notice to users that the WooCommerce AI plugin is being deprecated, if they interact with the plugin somehow.
+ */
+class Woo_AI_Deprecation_Notice {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		add_action( 'admin_footer', array( $this, 'output_deprecation_notice_data' ) );
+
+		// Register AJAX actions
+		add_action( 'wp_ajax_woo_ai_dismiss_deprecation', array( $this, 'dismiss_notice' ) );
+
+		// Check for reset query parameter for testing purposes
+		// TODO: Remove this before merging
+		if ( isset( $_GET['reset_woo_ai_notice'] ) && current_user_can( 'manage_options' ) ) {
+			delete_option( 'woo_ai_deprecation_dismissed' );
+			$current_url = remove_query_arg( 'reset_woo_ai_notice', $_SERVER['REQUEST_URI'] );
+			wp_safe_redirect( $current_url );
+			exit;
```

Comment:
Should we remove this code, right?

Reply:
Removed! I was leaving it there temporarily for testing.